acceptance/analyzer_test.go

```diff
@@ -979,13 +918,13 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 									ctrPath("/layers"),
 									analyzeImage,
 									h.WithFlags(
-										"--env", "CNB_REGISTRY_AUTH="+cacheAuthConfig,
+										"--env", "CNB_REGISTRY_AUTH="+fixtures.authRegAuthConfig,
```

Comment:
This is being defined with `appMeta` while before, `cacheAuthConfig` was defined with `cacheMeta`.

Reply:
If I'm not mistaken, we should use the second return value from line 140.

Reply:
Marking as solved following https://github.com/buildpacks/lifecycle/pull/668#discussion_r674152756.